app/controllers/container_controller.rb

```diff
@@ -111,6 +111,7 @@ def identify_container(id = nil)
 
   # ST clicked on in the explorer right cell
   def x_show
+    get_tagdata(Container.find_by_id(from_cid(params[:id])))
```

Comment:
@zeari this looks good, this controller does not go thru show method in mixin so we have to add call to get_tagdata explicitly in x_show here.